### PR TITLE
Update generator to twig 2.7

### PR DESCRIPTION
**Pull request by svandervlugt**

### CODE REVIEW

File: src/Twig/PerLineNode.php [hunk 1]
```diff
@@ -22,18 +25,18 @@
...
  *            the nodes between the prefix and/or postfix.
  */
-class PerLineNode extends \Twig_Node
+class PerLineNode extends Node
 {
     /**
      * Create new PerLineNode
      *
-     * @param \Twig_Node $lines
+     * @param Node $lines
```
Comment (ping-localhost) [inline src/Twig/PerLineNode.php]: Please align `$lines` just like the rest

File: composer.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
         "doctrine/inflector":   "~1.1",
         "doctrine/orm":         "~2.4",
         "symfony/filesystem":   "^3.0.2|~2.5|^4.0",
-        "twig/twig":            "^2.0||^1.28"
+        "twig/twig":            "~2.7"
```
Comment (ping-localhost) [inline composer.json]: `^2.7.0` and this means we need to do a major release.

**Review by ping-localhost — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (svandervlugt) [reply composer.json]: Disagree on ^2.7. Twig does not follow semver. 

Also from this package perspective this is not a BC-break. It just means that if you do not update twig, you can also not update this package.

Comment (ping-localhost) [reply composer.json]: It should follow semver (at least to from what I've seen/heard/read), but I cannot find anything to make a valid stance, so :+1:.

**Review by ping-localhost — APPROVED** [review]

File: composer.json (hunk 2 above)
Comment (haroldiedema) [reply composer.json]: Can't we keep the 1.x branch in here as well?

The security issue was fixed in 1.38.0.

So, `^2.7.0||^1.38.0` should be the correct one.

Comment (haroldiedema) [inline composer.json]: Why are we removing support for 1.x?

The security issue was fixed in both 1.38 and 2.7.

`^2.7.0||^1.38.0` should suffice.

**Review by haroldiedema — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (svandervlugt) [reply composer.json]: I was under the impression ^1.38 would not be namespaced. It is, so I will add it.

Will also use the `^`-notation, as it seems like the BC was unintentional.

**Review by ping-localhost — APPROVED** [review]

**Review by haroldiedema — APPROVED** [review]

**Pull request merged**